src/sqe.rs

```diff
@@ -63,9 +90,74 @@ impl<'ring> SubmissionQueue<'ring> {
     }
 }
 
+///
```

Comment:
I think these docs were supposed to be attached somewhere else? Possibly prep_fsync or SubmissionFlags?

Reply:
Sorry about this! I meant to stage another example instead of this one. I've removed this one for now until I get a better handle on read / write prep. The one I meant to add is at the `SubmissionQueue` struct under *Examples*.